visualization/babel.config.json

```diff
@@ -0,0 +1,15 @@
+{
+	"presets": [
+		[
+			"@babel/preset-env",
+			{
+				"targets": {
+					"node": "current"
```

Comment:
When do we actually transpile for the browser? I would have expected that to be the actual goal?

Reply:
That confuses me as well. I think we run node, when I start the standalone using nwjs.

I found this snipped here in the [documentation](https://babeljs.io/docs/en/6.26.3/babel-preset-env)
```json
{
	"presets": [
		[
			"@babel/preset-env",
			{
				"targets": {
					"node": "current",
					"browsers": ["last 2 versions"]
				},
				"corejs": 3,
				"useBuiltIns": "usage"
			}
		],
		"@babel/preset-typescript"
	]
}
```

This would polyfill for the latest two versions of all browser. I don't know what a good number here would be.
Any ideas?

Reply:
Let's check what support we had so far and use that. We could also define a support range. That would probably be useful. I guess the transpiration for the browser happens in the webcam settings if they are separate (I did not check)

Reply:
We never defined a support range, so you won't find one.

Reply:
I personally am fine to support the last 2 versions. IE 11 should not be supported anymore.

Reply:
I added a [browserslist](https://www.npmjs.com/package/browserslist) field to the package.json that is read by babel per default.
I'm not sure if I have to install the browserlist npm package though. There was no installation guide that implies that.